tools/localizations/cn.csv

```diff
@@ -34,4 +34,19 @@ Zigbee Switch,Zigbee开关
 Zigbee Thermostat,Zigbee温控
 Zigbee Thing,Zigbee设备
 Zigbee Vent,Zigbee新风系统
-Zigbee Window Treatment,Zigbee窗帘电机
\ No newline at end of file
+Zigbee Window Treatment,Zigbee窗帘电机
+Aqara Motion Sensor T1, Aqara 人体传感器 T1
+Aqara High Precision Motion Sensor, Aqara高精度人体传感器
+Aqara Door and Window Sensor T1, Aqara 门窗传感器T1
+Aqara Temperature and Humidity Sensor T1, Aqara温湿度传感器 T1
+Aqara Water Leak Sensor, Aqara 水浸传感器 T1
+Aqara Illuminance Sensor T1, Aqara 光照传感器T1
+Aqara Presence Sensor FP1, Aqara 人体存在传感器FP1
+Aqara Smart Dimmer Controller T1 Pro, Aqara 双色温驱动器 T1 Pro
+Aqara Single Switch Module T1 (With Neutral), Aqara 单路控制器T1 (零火版)
+Aqara Curtain Controller, Agara智能窗帘电机 (Zigbee开合帘版)
+Aqara Roller Shade Controller, Aqara智能管状电机
+Aqara Smart Pet Feeder C1, Aqara智能宠物喂食器 C1
+"Aqara Smart Wall Switch H1 (With Neutral, Single Rocker)", Aqara智能墙壁开关H1 (零火线单键版)
```

Comment:
Are these supposed to be surrounded with `"`?

Reply:
Yes. Because they have commas they trip up the csv reader. Luckily it's smart enough to handle quotes.